doc/man3/EVP_PKEY_copy_parameters.pod

```diff
@@ -46,6 +46,9 @@ Since OpenSSL private keys contain public key components too the function
 EVP_PKEY_eq() can also be used to determine if a private key matches
 a public key.
 
+EVP_PKEY_cmp() and EVP_PKEY_cmp_parameters() differ in their return values 
+compared to other _cmp() functions.
+
```

Comment:
You could add here something like "....cmp and cmp_parameters" functions differ in their return values 
compared to other _cmp() functions, they are just aliases for the _eq and parameters_eq functions provided for backwards compatibility with existing applications."